sdcm/nemesis.py

```diff
@@ -1523,6 +1523,79 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_nodetool_removenode(self):
+        """
+        https://docs.scylladb.com/operating-scylla/procedures/cluster-management/remove_node/
+
+        1.terminate
+        2.Full repair
+        3.Nodetool removenode
+        4.Sleep 10 minutes
+        5.Add new node
+        """
+
+        self._set_current_disruption('TerminateAndRemoveNodeMonkey')
+        node_to_remove = self.target_node
+        up_normal_nodes = self.cluster.get_nodes_up_and_normal(verification_node=node_to_remove)
+        node = random.choice(up_normal_nodes)
+        # node_to_remove must be different than node
+        while node is node_to_remove:
+            node = random.choice(up_normal_nodes)
```

Comment:
I don't understand why we need this?
target node should be provided already to the the disruption.

Reply:
once you remove a node, you need to call `nodetool remove` on other node.